src/cita_bft.rs

```diff
@@ -1766,6 +1766,7 @@ impl Bft {
                         }
                     }
                 }
+                _ => continue,
```

Comment:
Should we panic if the WAL file contains a unknown log type? That may indicate a corrupted file.
The WAL in `consensus_raft` only allows a corrupted file tail caused by interrupted IO.

Reply:
This can increase backward compatibility; for example, when a new type is added in a newer version, it works fine when using the old WAL file.